agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -242,6 +242,55 @@ if [ $lcl_cnt -gt 0 ]; then
 fi
 wait
 
+stockpile_path=$(getconf.py -C $configfile stockpile_path stockpile)
+stockpile_user=$(getconf.py -C $configfile stockpile_user stockpile)
+stockpile_local_remote_user=$(getconf.py -C $configfile local_remote_user stockpile)
+stockpile_host_remote_user=$(getconf.py -C $configfile host_remote_user stockpile)
+stockpile_output_path=$(getconf.py -C $configfile stockpile_output_path stockpile)
+
+# Install stockpile and its dependencies
+which git &>/dev/null
+if [[ $? != 0 ]]; then
+	debug_log "[$script_name] git package is not installed on the system, installing"
+	echo "git package is not installed on the systen, installing"
+	yum install git -y &>/dev/null
+fi
+which ansible &>/dev/null
+if [[ $? != 0 ]]; then
+	debug_log "[$script_name] ansibke package is not installed on the system, installing"
+	echo "ansible is not installed on the system, installing"
+	yum install ansible -y &>/dev/null
+fi
+if [[ -d "$stockpile_path/stockpile" ]]; then
+	rm -rf $stockpile_path/stockpile
+fi
+git clone https://github.com/redhat-performance/stockpile.git $stockpile_path/stockpile &>/dev/null
```

Comment:
Stockpile git repo is cloned for now to get the stockpile code. It would be great if we make a release in stockpile and package it as an rpm. The only problem with this approach is that the stockpile is very fast moving, so we won't get the latest changes.